### ISSUE TITLE
商务智能 HW02

### ISSUE THREAD

**Issue by HuangFuSL** [HuangFuSL/PhD2022-2023A#8]
Homework
1. write python code to learn Logistic regression model parameters based on gradient descent, using Iris.csv dataset as training dataset. let label Iris-setosa be the positive label and other labels be the negative label.
2. Draw a MLP model having two hidden layers with 3 and 2 nodes in the first and second hidden layer respectively. Suppose the input has 2 variables and the output has 3 class labels. There are three training examples with the three labels respectively. Write the forward computation process and backward propagation process. Hidden layers use Sigmoid activation function and final loss uses cross entropy loss.
